packages/server/src/routes/tenants.js

```diff
@@ -17,6 +17,28 @@ router.get('/', requireUser, async (req, res) => {
     res.json(response);
 });
 
+router.post('/', requireAdminUser, async (req, res) => {
+    if (!req.body.name || !req.body.agency_id) {
+        res.status(400).send('Tenant name and agency ID is required');
+        return;
+    }
+
+    try {
+        const tenant = {
+            display_name: req.body.name,
+            main_agency_id: req.body.agency_id,
+        };
+
+        const result = await createTenant(tenant);
```

Comment:
Should we be doing any validation/sanitization of the request data before sending it  to the DB, or is that built in somewhere else?

Reply:
That's a great idea - as far as I understand it there aren't any sort of validation built into our endpoints as is the case here with [users](https://github.com/usdigitalresponse/usdr-gost/blob/4389f12e9aa797b77d4102337d3480491c9efa27/packages/server/src/routes/users.js#L8) as well.

Since we don't have any validation or sanitation anywhere - what do you think about adding this endpoint for now to unblock https://github.com/usdigitalresponse/usdr-gost/pull/189 and then creating a separate issue where we add validation + sanitation to all of our endpoints?

Reply:
Makes sense to me.